build/images/Dockerfile.build.windows

```diff
@@ -44,3 +47,10 @@ COPY --from=antrea-build-windows  /antrea/bin/antrea-cni.exe /k/antrea/cni/antre
 
 RUN mkdir C:\k\antrea\utils
 COPY --from=antrea-build-windows /wins/wins.exe /k/antrea/utils/wins.exe
+
+COPY --from=windows-ovs /Windows/System32/vcruntime140.dll /Windows/System32/
+COPY --from=windows-ovs /Windows/System32/libeay32.dll /Windows/System32/
+COPY --from=windows-ovs /Windows/System32/ssleay32.dll /Windows/System32/
+
+RUN mkdir C:\openvswitch
+COPY --from=windows-ovs /openvswitch /openvswitch/
```

Comment:
Does it work if we only copy /openvswitch/usr/bin to antrea-agent image?

Reply:
Yes, Thanks @wenyingd